actions/restore

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -ex
+
+# This is an encapsulation of the "Restoring a Backup" section of the Etcd
+# administration guide located here: https://coreos.com/etcd/docs/latest/admin_guide.html
+#
+# Note: That restoration of a clusters data *must* be performed on a single etcd
+# cluster node, in a new state. Restoration of existing cluster state is not
+# supported at this time.
+
+ARCHIVE=etcd-data-$DATESTAMP.tar.gz
+DATESTAMP=$(date +%Y-%m-%d-%H.%M.%S)
```

Comment:
arg, yes. I alphabetized them while sleepy and didn't even think about bash ordering requirements. Good catch